python/TestHarness/TestHarness.py

```diff
@@ -576,7 +578,7 @@ def handleTestResult(self, specs, output, result, start=0, end=0, add_to_table=T
     # in the 'Final Test Results' area.
     if add_to_table:
       self.test_table.append( (specs, output, result, timing, start, end) )
-      if result.find('OK') != -1 or result.find('DRY_RUN') != -1:
+      if result.find(specs['status_message']) != -1 or result.find('DRY_RUN'):
```

Comment:
Instead of adding a third 'or', why not just search by the successful status message :)